lib/fetch/index.js

```diff
@@ -92,13 +92,13 @@ export function fetchTalk(args) {
 }
 
 // TODO: Test? lol
```

Comment:
If there's a problem with the code and this PR touches that part of the code, isn't it relevant to this PR?